gapic/api/artman_errorreporting.yaml

```diff
@@ -0,0 +1,28 @@
+common:
+  api_name: google-devtools-clouderrorreporting-v1beta1
+  import_proto_path:
+    - ${THISDIR}/../..
+  src_proto_path:
+    - ${THISDIR}/../../google/devtools/clouderrorreporting/v1beta1
+  service_yaml:
+    - ${THISDIR}/../../google/devtools/clouderrorreporting/errorreporting.yaml
+  gapic_api_yaml:
+    - ${THISDIR}/../../google/devtools/clouderrorreporting/v1beta1/clouderrorreporting_gapic.yaml
+  auto_merge: true
+  auto_resolve: true
+  ignore_base: false
+  output_dir: ${REPOROOT}/artman/output
+java:
+  final_repo_dir: ${REPOROOT}/gcloud-java/gcloud-java-errorreporting
+python:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-python-errorreporting
+go:
+  final_repo_dir: ${REPOROOT}/gapi-errorreporting-go
+csharp:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-csharp-errorreporting
+php:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-php-errorreporting
+ruby:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-ruby-errorreporting
```

Comment:
It's going to be a part of gcloud, then this will be `${REPOROOT}/gcloud-ruby/google-cloud-errorreporting` probably.